ecommerce/src/main/kotlin/io/hhplus/ecommerce/coupon/application/consumer/CouponIssueConsumer.kt

```
@@ -0,0 +1,105 @@
+
+            val payload = json.decodeFromString<CouponIssueRequestPayload>(record.value())
+
+            processCouponIssue(payload)
+
+            acknowledgment.acknowledge()
+
+            logger.info(
+                "[CouponIssueConsumer] 쿠폰 발급 완료: couponId={}, userId={}",
+                payload.couponId, payload.userId
+            )
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)